panc/src/main/scripts/panlint/panlint.py

```diff
@@ -33,6 +33,18 @@ def __init__(self, filename, number, text):
         self.filename = str(filename)
         self.number = int(number)
         self.text = str(text)
+        self.problems = []
+
+
+class Problem(object):
```

Comment:
Add the info about the purpose of this class from the commit log here as a docstring?